src/tea_ex.res

```diff
@@ -8,47 +8,56 @@ let renderEvent = (~key="", msg) => {
 }
 module LocalStorage = {
   open Tea_task
-  let length = nativeBinding(cb =>
-    switch Web.Window.LocalStorage.length(Web.Window.window) {
-    | None => cb(Error("localStorage is not available"))
-    | Some(value) => cb(Ok(value))
+
+  let length = nativeBinding(cb => {
+    try {
+      cb(Ok(Dom.Storage.length(Dom.Storage.localStorage)))
+    } catch {
+    | Not_found => cb(Error("localStorage is not available"))
```

Comment:
It's not clear to me why this is `Not_found`. I think any exception should go here.

Reply:
Same for all of these

Reply:
Do we know what other exceptions do we need to catch ?
Do we need to catch 
`| Invalid_argument(_) =>...`
 ( if yes, should I change the error message?)
Do we need to catch JS Exceptions using :
`| Js.Exn.Error(obj)`
even though they don't encourage that in the documentation.
Or is there a better way to catch all exceptions without writing them one by one?
I don't know if it is correct to do something like that
    `| _e => cb(Error("localStorage is not available"))`

Reply:
`| _e => cb(Error("localStorage is not available"))` - yeah, exactly